vuu-ui/packages/vuu-utils/src/screenshot-utils.ts

```diff
@@ -2,22 +2,29 @@ import { toPng } from "html-to-image";
 
 /**
  * Takes a screenshot of the given node and returns the base64 encoded image url
- * @param node Node to take screenshot of
+ * @param node HTMLElement to take screenshot of
  * @returns Base64 encoded image url
  */
-export async function takeScreenshot(node: HTMLElement) {
-
-  const screenshot = await toPng(node, { cacheBust: true })
-    .then((dataUrl) => {
-      return dataUrl;
+export const takeScreenshot = (node: HTMLElement): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    toPng(node, {
+      cacheBust: true,
+      filter: (child) =>
+        child.nodeType === Node.TEXT_NODE ||
+        child.getAttribute("role") !== "row",
```

Comment:
These look like good filter criteria to me, but I'd like to know what @heswell thinks.